modules/db/src/main/java/org/mule/module/db/internal/processor/AbstractSingleQueryDbMessageProcessor.java

```diff
@@ -43,4 +50,36 @@ protected Object executeQuery(DbConnection connection, MuleEvent muleEvent) thro
     }
 
     protected abstract Object doExecuteQuery(DbConnection connection, Query query) throws SQLException;
+
+    @Override
+    protected List<FieldDebugInfo> getMessageProcessorDebugInfo(DbConnection connection, MuleEvent muleEvent)
+    {
+        MuleEvent eventToUse = resolveSource(muleEvent);
+        final List<FieldDebugInfo> fields = new ArrayList<>();
+
+        Query resolvedQuery;
+        try
+        {
+            resolvedQuery = queryResolver.resolve(connection, eventToUse);
+        }
+        catch (QueryResolutionException e)
+        {
+            fields.add(createFieldDebugInfo(DbDebugInfoConstants.QUERY_DEBUG_FIELD, String.class, e));
+            return fields;
+        }
+
+        fields.add(createFieldDebugInfo(DbDebugInfoConstants.QUERY_DEBUG_FIELD, String.class, resolvedQuery.getQueryTemplate().getSqlText()));
+        fields.add(createFieldDebugInfo(DbDebugInfoConstants.TYPE_DEBUG_FIELD, String.class, resolvedQuery.getQueryTemplate().getType().toString()));
+
+        final List<FieldDebugInfo> paramFields = new ArrayList<>();
+        int paramIndex = 1;
+        for (QueryParamValue queryParamValue : resolvedQuery.getParamValues())
+        {
+            final String name = queryParamValue.getName() == null ? PARAM_DEBUG_FIELD_PREFIX + paramIndex++ : queryParamValue.getName();
+            paramFields.add(createFieldDebugInfo(name, String.class, queryParamValue.getValue()));
+        }
+        fields.add(createFieldDebugInfo(DbDebugInfoConstants.INPUT_PARAMS_DEBUG_FIELD, List.class, paramFields));
```

Comment:
Is there a reason to show the params names and values in a different way? Maybe an ordered map of name -> value can be better visualized.